src/WingetCreateCLI/Commands/NewCommand.cs

```diff
@@ -138,10 +139,18 @@ public override async Task<bool> Execute()
                 if (string.IsNullOrEmpty(this.OutputDir))
                 {
                     this.OutputDir = Directory.GetCurrentDirectory();
+                }
+
+                GitHub client = new GitHub(null, this.WingetRepoOwner, this.WingetRepo);
+                if (await client.CheckDuplicatePackageId(manifests.VersionManifest.PackageIdentifier))
+                {
+                    Console.WriteLine();
+                    Logger.ErrorLocalized(nameof(Resources.PackageIdAlreadyExists_Error));
+                    return false;
                 }
```

Comment:
Should we move this to be right after line 131?

Reply:
I thought about this earlier and I decided not to put it there because we still give the user an opportunity to go back and change their values and they could change it correctly the second time. If we were to put it right after line 131, that would mean the duplicate check only takes place after all of the fields from the VersionManifest have been prompted, which would end on default locale. This would seem odd to the user that the check happens after default locale and not packageId. This is just my thinking so if you feel strongly about it, I'd be okay with changing it.

Reply:
Talked about this offline. We agreed that prompting package ID should be separate from the other fields as this requires an additional duplicate check.